### PR TITLE
Document using `Deployer`

### PR DESCRIPTION
**Pull request by cptartur**
## Describe your changes

This PR add basic documentation of the `Deployer` module with an example usage.

## Linked issues

Closes

## Breaking changes

- [ ] This issue contains breaking changes

### CODE REVIEW

File: lib/starknet-jvm.md [hunk 1]
```diff
@@ -236,6 +236,27 @@ Data classes representing StarkNet objects.
 
 Data classes representing StarkNet transactions.
 
+# Package com.swmansion.starknet.deployercontract
+
+Classes for interacting with Universal Deployer Contract (UDC).
+
+```java
```
Comment (THenry14) [inline lib/starknet-jvm.md:243]: only java?

Comment (cptartur) [reply lib/starknet-jvm.md:243]: All other examples in package specific docs are java-only.

Comment (THenry14) [reply lib/starknet-jvm.md:243]: It's a mixed bag, there's kotlin example for StandardAccount. I think UDC is important enough to feature in kotlin examples.

Comment (cptartur) [reply lib/starknet-jvm.md:243]: Okay, I'll include a Kotlin example

**Pull request merged**